sematic/types/types/chip.py

```diff
@@ -0,0 +1,40 @@
+# Standard library
+# Standard Library
+from dataclasses import dataclass
+
+
+@dataclass
+class Chip:
```

Comment:
The issue (#191) stated that Python Enum values should show up as chips / pills in the UI, meaning that artifacts of Enum types should be rendered that way.

In order to use this new type, we would need to first convert the Enum values to this dataclass, which is an undesired intermediate step.

Instead of introducing this dataclass, could you please render Enums as chips?

Reply:
Yes, @tscurtu is correct here. We don't need to add a new Python type for this work. We simply need to add special treatment for `Enum` in the UI. This change should be only a front-end change.

Feel free to reach out to us on [Discord](https://discord.gg/4KZJ6kYVax) to discuss this PR.